### PR TITLE
Many things

### PR DESCRIPTION
**Pull request by trueworldfacts**
Prettier code formatter, update source.json, add segs command, and fix starboard

### CODE REVIEW

File: src/commands/segs.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      .setName("who")
+      .setDescription("Who/what/whatever to seg")
+      .setRequired(true)
+  );
+
+let _;
+export default _ = {
+  process: run,
+  command: cmd,
+};
```
Comment (KingRainbow44) [inline src/commands/segs.ts:25]:
```suggestion
```

**Review by KingRainbow44 — CHANGES_REQUESTED** [review]
delete

File: src/commands/segs.ts (hunk 1 above)
Comment (trueworldfacts) [reply src/commands/segs.ts:25]: It's a necessary command needed to make the bot fun and interactive.

**Review by memetrollsXD — COMMENTED** [review]
what in the commit changes... make a PR for every feature separately

**Comment by trueworldfacts** [comment]
The PR description tells you smh

**Pull request merged**